Source/WebCore/inspector/agents/InspectorPageAgent.cpp

```diff
@@ -681,6 +686,8 @@ static std::optional<Cookie> parseCookieObject(Inspector::Protocol::ErrorString&
         return std::nullopt;
     }
 
+    cookie.partitionKey = cookieObject->getString("partitionKey"_s);
```

Comment:
NIT: since this is optional we should probably do it last so that we don't do unnecessary work if something else fails